src/odinapi/templates/dataaccess.html

```diff
@@ -377,19 +377,76 @@ <h2>Recommended projects</h2>
 <p>
 Note that all spaces in the product names need to be replaced by
 <code>%20</code> in the request.
+</p>
+<p>
+For a detailed discussion of the different products, see the
+<a href="{{ url_for('static', filename='documents/PVER.pdf') }}">Odin/SMR Product Validation and Evolution Report</a>.
 </p>
           <div>
             <table class="table">
               <tbody>
                 <tr>
-                  <td><b>Species:</b></td>
-                  <td><b>Project:</b></td>
-                  <td><b>Product name:</b></td>
+                  <td><b>Species</b></td>
+                  <td><b>Project</b></td>
+                  <td><b>Product name</b></td>
+                  <td><b>Vertical coverage<b></td>
+                  <td><b>Comment</b></td>
                 </tr>
                 <tr>
-                  <td>Stratospheric ozone</td>
+                  <td>Ozone</td>
                   <td>ALL-Strat-v3.0.0</td>
                   <td>O3 / 545 GHz / 20 to 85 km</td>
```

Comment:
this is our internal name of the product right? Otherwise I would exepct subscript on the 3

Reply:
yes